src/posix/enumerate.rs

```diff
@@ -139,7 +139,6 @@ fn port_type(d: &libudev::Device) -> Result<SerialPortType> {
                 d.property_value("ID_USB_MODEL_ID"),
                 d.property_value("ID_USB_VENDOR"),
                 d.property_value("ID_USB_MODEL"),
```

Comment:
`ID_USB_VENDOR` and `ID_USB_MODEL` are optional with respect to building `UsbPortInfo` like `ID_USB_SERIAL_SHORT`. Shouldn't they be excluded from the "criterion vector" as well?

Reply:
Your're right. These two lines should also be deleted.